Makefile

```diff
@@ -66,5 +66,12 @@ pr:
 	OPAMYES=1 opam publish submit $(NAME).$(VERSION) && rm -rf $(NAME).$(VERSION)
 
 ppx:
+	./configure
```

Comment:
But OK, agree that it should only be called when needed, just did not find an easy way to do it.